### DISCUSSION TITLE
ScriptReference/EditorGUI-indentLevel

### DISCUSSION THREAD

**Discussion by giscus** [Developer-Notes-Extension/Unity-Notes#40] [Notes]
# ScriptReference/EditorGUI-indentLevel

https://docs.unity3d.com/ScriptReference/EditorGUI-indentLevel.html

**Comment by vertxxyz** [comment]
See [`IndentLevelScope`](https://docs.unity3d.com/ScriptReference/EditorGUI.IndentLevelScope.html) for a robustly managed indent area.
There is also an undocumented function: `Rect EditorGUI.IndentedRect(Rect source)` that will return an indented rect.

**Comment by andreasIsCoding** [comment]
The width of a single indent is a hard-coded 15 pixels.  From EditorGui.cs:
`private const float kIndentPerLevel = 15;`
